CoreFoundation/CMakeLists.txt

```diff
@@ -27,8 +27,11 @@ find_package(Threads REQUIRED)
 if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
   find_package(LibXml2 REQUIRED)
   find_package(CURL CONFIG)
-  if(NOT CURL_FOUND)
+  if(CURL_FOUND)
+    set(NS_CURL_VERSION ${CURL_VERSION})
+  else()
     find_package(CURL REQUIRED)
+    set(NS_CURL_VERSION ${CURL_VERSION_STRING})
```

Comment:
I’m not sure I see the value in the NS prefix here.  What’s the motivation for that?

Reply:
Nothing special actually. Mostly inspired by NS_CURL_ASSUME_FEATURES_MISSING, NS_CURL_MISSING_MAX_HOST_CONNECTIONS, etc. Any suggestions are appreciated.

Reply:
The difference is that `CURL_VERSION` and `CURL_VERSION_STRING` is that one is a string representation I believe.  I think that we invert the case where the `CONFIG` check just does a `set(CURL_VERSION_STRING "${CURL_VERSION}")` would be simpler.

Reply:
I didn't want to define option originated from external module. But why not. As it really makes everything simpler. Will do.